<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>VOiD Set Wallpaper</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../Downloads/VOiD-icon.png</normaloff>../../Downloads/VOiD-icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>2</number>
        </property>
        <widget class="QWidget" name="tab_set">
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <attribute name="title">
          <string>Set wallpaper</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QFormLayout" name="formLayout_4">
            <property name="fieldGrowthPolicy">
             <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
            </property>
            <property name="rowWrapPolicy">
             <enum>QFormLayout::DontWrapRows</enum>
            </property>
            <property name="labelAlignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="formAlignment">
             <set>Qt::AlignHCenter|Qt::AlignTop</set>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <item row="0" column="0">
             <widget class="QPushButton" name="pushButton_tab1_open_file">
              <property name="text">
               <string>Open file</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="lineEdit_tab1_file_path">
              <property name="enabled">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_tab1_wallpaper_style">
              <property name="text">
               <string>Wallpaper Style</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QRadioButton" name="radioButton_tab1_center">
                <property name="text">
                 <string>Center</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="radioButton_tab1_tile">
                <property name="text">
                 <string>Tile</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="radioButton_tab1_stretch">
                <property name="text">
                 <string>Stretch</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="radioButton_tab1_fit">
                <property name="text">
                 <string>Fit</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="radioButton_tab1_fill">
                <property name="text">
                 <string>Fill</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="radioButton_tab1_span">
                <property name="text">
                 <string>Span</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_tab1_theme">
              <property name="text">
               <string>Theme</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QRadioButton" name="radioButton_tab1_light">
                <property name="text">
                 <string>Light</string>
                </property>
                <attribute name="buttonGroup">
                 <string notr="true">buttonGroup_tab1_theme</string>
                </attribute>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="radioButton_tab1_dark">
                <property name="text">
                 <string>Dark</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
                <attribute name="buttonGroup">
                 <string notr="true">buttonGroup_tab1_theme</string>
                </attribute>
               </widget>
              </item>
             </layout>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_tab1_wallpaper_preview">
              <property name="text">
               <string>Wallpaper Preview</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLabel" name="label_tab1_image_preview">
              <property name="text">
               <string>Image Preview</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_tab1_setwallpaper">
            <property name="text">
             <string>Set Wallpaper</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_get">
         <attribute name="title">
          <string>Get wallpaper info</string>
         </attribute>
         <layout class="QFormLayout" name="formLayout">
          <property name="labelAlignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="formAlignment">
           <set>Qt::AlignHCenter|Qt::AlignTop</set>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label_tab2_wallpaper_path">
            <property name="text">
             <string>Wallpaper path</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineEdit_tab2_file_path">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_tab2_wallpaper_style">
            <property name="text">
             <string>Wallpaper style</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QRadioButton" name="radioButton_tab2_center">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Center</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButton_tab2_tile">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Tile</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButton_tab2_stretch">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Stretch</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButton_tab2_fit">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Fit</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButton_tab2_fill">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Fill</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButton_tab2_span">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Span</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_tab2_wallpaper_preview">
            <property name="text">
             <string>Wallpaper preview</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLabel" name="label_tab2_image_preview">
            <property name="text">
             <string>Wallpaper preview</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_tab2_theme">
            <property name="text">
             <string>Theme</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QRadioButton" name="radioButton_tab2_light">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Light</string>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">buttonGroup_tab2_theme</string>
              </attribute>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButton_tab2_dark">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Dark</string>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">buttonGroup_tab2_theme</string>
              </attribute>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Logs</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QPlainTextEdit" name="plainTextEdit_log">
            <property name="acceptDrops">
             <bool>false</bool>
            </property>
            <property name="undoRedoEnabled">
             <bool>false</bool>
            </property>
            <property name="lineWrapMode">
             <enum>QPlainTextEdit::NoWrap</enum>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup_tab1_theme"/>
  <buttongroup name="buttonGroup_tab2_theme"/>
 </buttongroups>
</ui>
